test/unit/HTMLBuilder.test.ts

```diff
@@ -196,4 +297,12 @@ describe('HTMLBuilder', () => {
     const results = calculateClauseCoverage(statementResults, clauseResults);
     expect(results).toEqual('100');
   });
+
+  test('html generation orders population first, then other, then function', () => {
+    const expectedHTML = getHTMLFixture('PopRetrieveFunc.html').replace(/\s/g, '');
+    const res = generateHTML(singlePopMeasure, [popRetrieveFuncElm], prfStatementResults, prfClauseResults, 'test');
+    expect(res.replace(/\s/g, '')).toEqual(expectedHTML);
```

Comment:
@mgramigna thoughts on ditching this expect statement and the `PopRetrieveFunc.html` fixture or worth keeping?

Reply:
yeet it outta there!

Reply:
yeeting...